Dockerfile.ci

```diff
@@ -1,17 +1,24 @@
 ARG commit=latest
 FROM elifesciences/proofreader-php:latest AS proofreader
 FROM elifesciences/hypothesis-dummy:${commit}
+ENV DEBUG=1
 
-# extract into another image and put here with multi-stage build
+# TODO: extract into a base image, but smoke tests should be in the prod image
 USER root
 RUN git clone https://github.com/asm89/smoke.sh /opt/smoke.sh
 
 USER elife
 COPY --from=proofreader --chown=elife:elife /srv/proofreader-php /srv/proofreader-php
 RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
 
-RUN composer install
-COPY --chown=elife .php_cs project_tests.sh smoke_tests.sh ${PROJECT_FOLDER}/
+RUN composer-install
+COPY --chown=elife test/ ${PROJECT_FOLDER}/test
+COPY --chown=elife .php_cs project_tests.sh phpunit.xml.dist smoke_tests.sh ${PROJECT_FOLDER}/
+RUN composer-post
+# TODO: extract into base image
```

Comment:
What we can extract into the base image is a script, that would be executed as `root` (as the `/srv/$project` directory doesn't exist yet in the base image). Being this a fast operation the main concern for where to put it could be where there is a `USER root` statement already.

Reply:
Or `COPY` the distributed folder and put it with the other files? (Assuming we still commit an empty `build` folder?)

Reply:
We may commit it to make host-based usage simpler, but you don't want to bake its contents (with random/host-dependent ownership and permissions) into the image as a new test run should always start with an empty folder.